devnotes/mvpd_generation.md

```diff
@@ -0,0 +1,40 @@
+# MVPD data modification
+
+## Test goals
+
+The main goal of this test is to check if `MVPD` partition is modified
+before starting hosboot by e.g. `HBBL` or some other part of software
+started by `Hostboot`. It is already known that `Hostboot` is not
```

Comment:
@IgorBagnucki Is it really?

Reply:
It's what I remember from our meeting and @krystian-hebel recently mentioned in his comment https://jira-3mdeb.atlassian.net/browse/IOP-158?focusedCommentId=63607
If I'm wrong, let me know, and I'll correct the information.

Reply:
I have updated the document and removed this assumption.